python/hsfs/engine/spark.py

```diff
@@ -492,14 +492,17 @@ def _random_split(self, dataset, training_dataset):
     def _time_series_split(
         self, training_dataset, dataset, event_time, drop_event_time=False
     ):
+        def _check_event_time_type(event_time):
+            # for backward compatibility
+            if isinstance(event_time, int) and len(str(event_time)) == 13:
+                event_time = int(event_time / 1000)
+            return util.convert_event_time_to_timestamp(event_time)
+
+        # registering the UDF
+        _convert_event_time_to_timestamp = udf(_check_event_time_type, LongType())
```

Comment:
can use `util.convert_event_time_to_timestamp` directly here.

Reply:
Well I don't want to put this part in `util.convert_event_time_to_timestamp`

```
            # for backward compatibility
            if isinstance(event_time, int) and len(str(event_time)) == 13:
                event_time = int(event_time / 1000)
```

Reply:
in util.py, we have
```        
    elif isinstance(event_time, int):
        if event_time == 0:
            raise ValueError("Event time should be greater than 0.")
        # jdbc supports timestamp precision up to second only.
        if len(str(event_time)) < 13:
            event_time = event_time * 1000
        return event_time
```
so, below is redundant.
```
if isinstance(event_time, int) and len(str(event_time)) == 13:
                event_time = int(event_time / 1000)
```

Reply:
honestly this is wrong:
```
        if len(str(event_time)) < 13:
            event_time = event_time * 1000
```

but I don't know how to replace without braking API. 

If we don't use this then how do we guarantee that even_time will have seconds granularity?

```
if isinstance(event_time, int) and len(str(event_time)) == 13:
                event_time = int(event_time / 1000)
```

Reply:
I can check either 10 or 13?